mungegithub/mungers/publisher_logger.go

```diff
@@ -64,3 +91,36 @@ func (p *plog) ReadLog() string {
 func (p *plog) Flush() {
 	glog.Flush()
 }
+
+func prefixFollowingLines(p, s string) string {
+	lines := strings.Split(s, "\n")
+	for i := range lines {
+		if i != 0 && len(lines[i]) > 0 {
+			lines[i] = p + lines[i]
+		}
+	}
+	return strings.Join(lines, "\n")
+}
+
+type cmdStr exec.Cmd
+
+func (cs cmdStr) String() string {
+	args := make([]string, len(cs.Args))
+	for i, s := range cs.Args {
+		if strings.IndexRune(s, ' ') != -1 {
+			args[i] = fmt.Sprintf("%q", s)
+		} else {
+			args[i] = s
+		}
+	}
+	return strings.Join(args, " ")
+}
+
+type muxWriter struct {
+	A, B io.Writer
+}
+
+func (w *muxWriter) Write(b []byte) (int, error) {
+	w.A.Write(b)
```

Comment:
nit: Check error here and return early if not nil?

Reply:
done